lib/src/ruby/module.cc

```diff
@@ -155,8 +149,7 @@ namespace facter { namespace ruby {
         ruby.rb_gc_register_address(&_on_message_block);
 
         // Install a logging message handler
-        on_message([this](log_level level, string const& message) {
-            auto const& ruby = *api::instance();
```

Comment:
Based on my thought-process there, we should still call api::instance within the lambda, just in-case.